ubuntudesign/www.ubuntu.com
# Core takeover

## Done

New homepage takeover 

## QA

Check the homepage takeover, visually, and compare to the [copydoc](https://docs.google.com/document/d/10e7MDM58mye49dVFXuLgNs4NESxy22v1ssRlUALGK9A/edit?ts=5810e1a2#)
